frontend/tasks/content.js

```diff
@@ -22,6 +22,17 @@ gulp.task('content-experiments-data', function generateStaticAPITask() {
     .pipe(gulp.dest(config.DEST_PATH));
 });
 
+// The next two tasks are hacks to copy the en-US locale 
+// to an en locale. See issue #2006 for background.
+gulp.task('content-watch-en', () => {
+   gulp.watch('./locales/en-US/**/*', ['content-build-en']);
+});
+
+gulp.task('content-build-en', () => {
+  return gulp.src('./locales/en-US/**/*')
+    .pipe(gulp.dest('./locales/en'));
```

Comment:
So we're copying ./locales/en-US/\* into ./locales/en/\*, but I see that we have a bunch of committed locales/en/*.{properties,ftl} files below. Since we're copying these assets in the `gulp build` step, should these files be committed to the repo?